package.json

```diff
@@ -21,6 +21,7 @@
   },
   "devDependencies": [
     "sass-lint",
+    "eslint",
```

Comment:
This appears to have been installed implicitly;
adding it explicitly here to pull in later security fixes.